yt/data_objects/static_output.py

```diff
@@ -221,7 +233,7 @@ def __init__(
         filename: str,
         dataset_type: Optional[str] = None,
         units_override: Optional[Dict[str, str]] = None,
-        unit_system: str = "cgs",
+        unit_system: Literal["cgs", "mks", "code"] = "cgs",
```

Comment:
as a note: are these really the only ones we allow?  I'm not sure that's the case.

Reply:
from what I can tell we actually rely on `unyt` to define `unyt.unit_systems.unit_system_registry`, and the value used here needs to be a (case insensitive) key of this dictionary (unless it's `"code"`). At runtime, this dict has the following keys:
`dict_keys(['cgs', 'mks', 'imperial', 'galactic', 'solar', 'geometrized', 'planck'])`

so you're right that there are more than 3 valid options. Unfortunately we can't define a `Literal` type programmatically which means our options are as follow:
- expand this type by hardcoding every current options, at the risk of it getting out of sync whenever unyt adds or removes one.
- get back to a more generic type (`str`), which is more future proof but doesn't convey much information.

I'd go with the first option because I don't think it's very likely that we change those in unyt any time soon, but I can settle for the second one too !

Reply:
Let's do option 1

Reply:
done !